frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -250,6 +250,17 @@ This is used to detect if the cover is closed, which will automatically sleep an
     }
 end
 
+if Device:isRemarkable() and PowerD:hasHallSensor() then
+    common_settings.cover_events = {
+        text = _("Disable reMarkable cover events"),
```

Comment:
@yparitcher Do you recall if there's a particular reason you went for `Disable Kindle cover events`?

In any case, at a quick glance it'd make an awful lot more sense to change this string to `Disable cover events` and to unify the two.

Reply:
Mostly because that was he only supported platform at the time.
but yes, definitely merge them to one menu entry

Reply:
If so `Powerd:hasHallSensor()` should be copied to `generic/powerd.lua` with a default of false:
```lua
function BasePowerD:hasHallSensor() return false end
```
and simplify this check.